diesel_cli/src/config.rs

```diff
@@ -0,0 +1,41 @@
+use clap::ArgMatches;
+use std::env;
+use std::error::Error;
+use std::fs;
+use std::io::Read;
+use std::path::PathBuf;
+use toml;
+
+use super::{find_project_root, handle_error};
+
+#[derive(Deserialize)]
+pub struct Config {
+    #[serde(default)]
+    pub print_schema: PrintSchema,
+}
+
+impl Config {
+    pub fn file_path(matches: &ArgMatches) -> PathBuf {
+        matches
+            .value_of("CONFIG_FILE")
+            .map(PathBuf::from)
+            .or_else(|| env::var_os("DIESEL_CONFIG_FILE").map(PathBuf::from))
+            .unwrap_or_else(|| {
+                find_project_root()
+                    .unwrap_or_else(handle_error)
+                    .join("diesel.toml")
```

Comment:
Does this create `diesel.toml` in the project root if it doesn't already exist?

Reply:
No. https://github.com/diesel-rs/diesel/pull/1630/files/ae22270600aeddeb8b95e6b88f48060bd514ab9a#diff-15619a8ba441d58f7250a6aa4a4b5273R200 does.